openapi-v1.yaml

```diff
@@ -6322,59 +6541,240 @@ paths:
           description: pagination limit
           type: integer
           required: false
+        - name: search
+          in: query
+          description: search string that will look at name, namespace or description fields
+          required: false
+          type: string
+        - name: namespace
+          in: query
+          description: namespace
+          required: false
+          type: string
+        - name: orderby
+          in: query
+          description: sort by which field valid values include last_accessed, size, name
+          required: false
+          type: string
+        - name: permissions
```

Comment:
@alekshs didn't we agree to drop filtering by permissions?

Reply:
I don't remember dropping permissions. If this is the case the backend also is simplified a lot. @Shelnutt2 can you clarify? Also we drop them for all domains or only owned?